### ISSUE TITLE
Problem with grid and chart visualization

### ISSUE THREAD

**Issue by pshpkr** [projectmesa/mesa#612]
Hello every one, I'm a student studying ABMS using mesa! Also, this is my first issue in the GitHub.  Please tell me if I made a mistake.

I am currently making a model using a multi-grid. However, when I increased the grid size, the grid module and the chad module overlap each other. I thought this was a problem for my code, but the same problem occurred in the money model example. I really want to know how can I fix it.
I need help. Thanks for reading!

**Describe the bug**
When I raised the grid size from 500 * 500 to 1000 * 1000 in the money model example, the following problems occurred.

**Expected behavior**
I expected the chart module to go under the grid. 

**To Reproduce**

![image](https://user-images.githubusercontent.com/5234756/50469226-d01def80-09ee-11e9-8d49-56e451e5f9ee.png)

**Additional context**
 I'm using Google Chrome and Spyder in Windows 10. Also, I'm using the Mesa 0.8.5

**Comment by dmasad** [comment]
Good catch! The good news is that you didn't do anything wrong; I'm replicating the problem on my end. The bad news is that I'm not sure how to solve it yet. I'll look into it -- and if anyone else can see the issue, please jump in!

**Comment by JamesHovet** [comment]
@pshpkr Just made a PR to hopefully solve this. Check out https://github.com/projectmesa/mesa/pull/615 if you want to use the fix for the project you are working on.

**Comment by pshpkr** [comment]
@JamesHovet Thanks a lot for your help! It works very well!

**Comment by Corvince** [comment]
Thanks a lot for reporting the issue!
